ocs_ci/ocs/ui/validation_ui.py

```diff
@@ -0,0 +1,155 @@
+import logging
+
+from ocs_ci.ocs.ui.base_ui import PageNavigator
+from ocs_ci.ocs.ui.views import locators
+from ocs_ci.utility.utils import get_ocp_version, TimeoutSampler
+from ocs_ci.framework import config
+from ocs_ci.ocs import constants
+
+
+logger = logging.getLogger(__name__)
+
+
+class ValidationUI(PageNavigator):
+    """
+    User Interface Validation Selenium
+
+    """
+
+    def __init__(self, driver):
+        super().__init__(driver)
+        ocp_version = get_ocp_version()
+        self.err_list = list()
+        self.validation_loc = locators[ocp_version]["validation"]
+
+    def verify_object_service_page(self):
+        """
+        Verify Object Service Page UI
+
+        """
+        self.navigate_overview_page()
+        self.do_click(self.validation_loc["object_service_tab"])
+        platform = config.ENV_DATA.get("platform").lower()
+        if platform == constants.VSPHERE_PLATFORM:
+            logger.info("Click on Object Service button")
+            self.do_click(self.validation_loc["object_service_button"])
+            logger.info("Click on Data Resiliency button")
+            self.do_click(self.validation_loc["data_resiliency_button"])
+        strings_object_service_tab = ["Total Reads", "Total Writes"]
+        self.verify_page_contain_strings(
+            strings_on_page=strings_object_service_tab, page_name="object_service"
+        )
+
+    def verify_persistent_storage_page(self):
+        """
+        Verify Persistent Storage Page
+
+        """
+        self.navigate_overview_page()
+        self.do_click(self.validation_loc["persistent_storage_tab"])
+        strings_object_service_tab = [
+            "Used Capacity",
+            "IOPS",
+            "Latency",
+            "Throughput",
+            "Recovery",
+            "Utilization",
+            "Used Capacity Breakdown",
+            "Raw Capacity",
+        ]
+        self.verify_page_contain_strings(
+            strings_on_page=strings_object_service_tab, page_name="persistent_storage"
+        )
```

Comment:
@shyRozen I understand your suggestion to search patterns (regular expression).
It is more complicated.I will  think on a solution..

Reply:
I think we could add the parameters a bit later. We could get the relevant numbers from the CLI and then check that the UI shows them correctly, but I think it would be better to make it a separate PR.